ami-build/scripts/common-fun.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -eu
+
+set -o pipefail
+
+
+AZP_USER=azure-pipelines
+GH_USER=runner
+
+
+_run_as () {
+    tmpexec="$(mktemp)"
+    chmod +x "$tmpexec"
```

Comment:
not sure if `${1}` will be able to read/exec here - probs more specific creds would be good